### PR TITLE
Added russian localization

### PR DESCRIPTION
**Pull request by macasasa**
Added russian localization for MainTBC,
Added russian localization for CustomSearchBar (Cancel button)

### CODE REVIEW

File: Say Their Names/Source/Components/CustomSearchBar.swift [hunk 1]
```diff
@@ -25,7 +25,7 @@ class CustomSearchBar: UIView {
     }()
     private let cancelSearchButton: UIButton = {
         let button = UIButton()
-        button.setTitle("Cancel", for: .normal)
+        button.setTitle(L10n.cancel, for: .normal)
```
Comment (Kilo-Loco) [inline Say Their Names/Source/Components/CustomSearchBar.swift:28]:
```suggestion
        button.setTitle(Strings.cancel, for: .normal)
```

File: Say Their Names/Source/Components/MainTabBarController.swift [hunk 2]
```diff
@@ -83,22 +83,22 @@ class MainTabBarController: UITabBarController, UITabBarControllerDelegate {
             
         homeNC.isNavigationBarHidden = true
         homeNC.tabBarItem.image = #imageLiteral(resourceName: "gallery")
-        homeNC.tabBarItem.title = "Home"
+        homeNC.tabBarItem.title = L10n.home
```
Comment (Kilo-Loco) [inline Say Their Names/Source/Components/MainTabBarController.swift:86]:
```suggestion
        homeNC.tabBarItem.title = Strings.home
```

File: Say Their Names/Source/Components/MainTabBarController.swift [hunk 3]
```diff
@@ -83,22 +83,22 @@ class MainTabBarController: UITabBarController, UITabBarControllerDelegate {
...
         
         donationsNC.isNavigationBarHidden = true
         donationsNC.tabBarItem.image = #imageLiteral(resourceName: "heart")
         donationsNC.tabBarItem.selectedImage = UIImage(named: "heart_active")
-        donationsNC.tabBarItem.title = "Donations"
+        donationsNC.tabBarItem.title = L10n.donations
```
Comment (Kilo-Loco) [inline Say Their Names/Source/Components/MainTabBarController.swift:91]:
```suggestion
        donationsNC.tabBarItem.title = Strings.donations
```

File: Say Their Names/Source/Components/MainTabBarController.swift [hunk 4]
```diff
@@ -83,22 +83,22 @@ class MainTabBarController: UITabBarController, UITabBarControllerDelegate {
...
                
         petitionsNC.isNavigationBarHidden = true
         petitionsNC.tabBarItem.image = #imageLiteral(resourceName: "petition")
         petitionsNC.tabBarItem.selectedImage = UIImage(named: "petition_active")
-        petitionsNC.tabBarItem.title = "Petitions"
+        petitionsNC.tabBarItem.title = L10n.petitions
```
Comment (Kilo-Loco) [inline Say Their Names/Source/Components/MainTabBarController.swift:96]:
```suggestion
        petitionsNC.tabBarItem.title = Strings.petitions
```

File: Say Their Names/Source/Components/MainTabBarController.swift [hunk 5]
```diff
@@ -83,22 +83,22 @@ class MainTabBarController: UITabBarController, UITabBarControllerDelegate {
...
         
         settingsNC.isNavigationBarHidden = true
         settingsNC.tabBarItem.image = #imageLiteral(resourceName: "settings")
         settingsNC.tabBarItem.selectedImage = UIImage(named: "settings_active")
-        settingsNC.tabBarItem.title = "Settings"
+        settingsNC.tabBarItem.title = L10n.settings
```
Comment (Kilo-Loco) [inline Say Their Names/Source/Components/MainTabBarController.swift:101]:
```suggestion
        settingsNC.tabBarItem.title = Strings.settings
```

**Comment by macasasa** [comment]
Thank you Kilo-Loco, i completely agree!

**Pull request merged**